library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Agreed.  Could we not just make additional functions in the headers file that allow you to load a list?

`function include_allthe_bootstrap_stuff_library($arg0,$arg1,$arg2,$arg3,$arg4,$arg5,$arg6,$arg7,$arg8,$arg9) `